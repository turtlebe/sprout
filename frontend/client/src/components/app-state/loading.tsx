import { LinearProgress } from '@plentyag/brand-ui/src/material-ui/core';
import React from 'react';

export const Loading: React.FC = () => <LinearProgress />;
