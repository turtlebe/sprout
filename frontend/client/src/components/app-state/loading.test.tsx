import { render } from '@testing-library/react';
import React from 'react';

import { Loading } from './loading';

describe('Loading', () => {
  it('renders', () => {
    const { asFragment } = render(<Loading />);
    expect(asFragment()).toMatchSnapshot();
  });
});
