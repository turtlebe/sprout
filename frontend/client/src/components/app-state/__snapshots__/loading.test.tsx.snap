// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders 1`] = `
<DocumentFragment>
  <div
    class="MuiLinearProgress-root MuiLinearProgress-colorPrimary MuiLinearProgress-indeterminate"
    role="progressbar"
  >
    <div
      class="MuiLinearProgress-bar MuiLinearProgress-barColorPrimary MuiLinearProgress-bar1Indeterminate"
    />
    <div
      class="MuiLinearProgress-bar MuiLinearProgress-bar2Indeterminate MuiLinearProgress-barColorPrimary"
    />
  </div>
</DocumentFragment>
`;
